### PR TITLE
Expand on the install tutorial

### LINKED ISSUE #61: Create User Docs - Tutorials/get-started
Following https://documentation.divio.com/tutorials/, add Hypper tutorials that tells end-users how to:

- [ ] Install Hypper
- [x] Use Hypper to install, upgrade, and otherwise work with charts
...

### PR DESCRIPTION
**Pull request by Itxaka**
Fixes #61 

Signed-off-by: Itxaka <igarcia@suse.com>

### CODE REVIEW

File: docs/user/tutorials/installing.md [hunk 1]
```diff
@@ -1,11 +1,49 @@
 # Installing Hypper
 
-## From source (Linux, macOS)
+This guide shows how to install the Hypper CLI. Hypper can be installed either from source, or from pre-built binary releases.
+
+
+## From the Binary Releases (Linux, macOS, windows)
```
Comment (viccuad) [inline docs/user/tutorials/installing.md:6]: Nitpicking:

```suggestion
## From Binary Releases (Linux, macOS, Windows)
```

File: docs/user/tutorials/installing.md [hunk 2]
```diff
@@ -1,11 +1,49 @@
...
+
+Every [release](https://github.com/rancher-sandbox/hypper/releases) of Hypper provides binary releases for a variety of OSes. These binary versions can be manually downloaded and installed.
```
Comment (viccuad) [inline docs/user/tutorials/installing.md:8]: Nit:

```suggestion
Every [release](https://github.com/rancher-sandbox/hypper/releases) of Hypper provides binaries for a variety of OSes. These binary versions can be manually downloaded and installed.
```

To reduce repetition.

File: docs/user/tutorials/installing.md [hunk 3]
```diff
@@ -1,11 +1,49 @@
...
+
+You can fetch that script, and then execute it locally. It's well documented so that you can read through it and understand what it is doing before you run it.
+
+```
+$ curl -fsSL -o get_hypper https://raw.githubusercontent.com/rancher-sandbox/hypper/main/scripts/get-hypper
+$ chmod 700 get_hypper
+$ ./get-hypper
+```
+
+Yes, you can curl https://raw.githubusercontent.com/rancher-sandbox/hypper/main/scripts/get-hypper | bash if you want to live on the edge.
```
Comment (viccuad) [inline docs/user/tutorials/installing.md:29]:
```suggestion
Yes, you can `curl https://raw.githubusercontent.com/rancher-sandbox/hypper/main/scripts/get-hypper | bash` if you want to live on the edge.
```

File: docs/user/tutorials/installing.md [hunk 4]
```diff
@@ -1,11 +1,49 @@
...
+
+---
+**NOTE**
+
+For installing using the script you need bash, curl/wget, and tar (openssl and sudo are optional).
+Please check how you can install those requisites in your OS if needed. 
+
+---
+
+## From source (Linux, macOS, windows)
```
Comment (viccuad) [inline docs/user/tutorials/installing.md:39]: Nit:

```suggestion
## From source (Linux, macOS, Windows)
```

File: docs/user/tutorials/installing.md [hunk 5]
```diff
@@ -1,11 +1,49 @@
...
 
 Building Hypper from source is a bit more work, but the best way to test the
 latest (pre-release) Hypper version.
 
-You must have a working Go environment (1.14+).
+You must have a working Go environment (1.14+), and a POSIX environment with the proper tools installed (git, make).
+
+For windows, Hypper has been tested under [cygwin](https://www.cygwin.com/) and it compiles correctly.
```
Comment (viccuad) [inline docs/user/tutorials/installing.md:46]:
```suggestion
For Windows, Hypper has been tested under [cygwin](https://www.cygwin.com/) and it compiles correctly.
```

File: docs/user/tutorials/installing.md [hunk 6]
```diff
@@ -15,17 +53,29 @@ $ make
...
+It will then compile `hypper`, and place it in `bin/hypper` (`bin/hypper.exe` for windows).
 
 
 ## From development builds
 
 In addition to releases you can download and install development snapshots of
 Hypper from [Github Actions].
 
-## From Github releases
+On each successful CI run, artifacts are stored for that run. Just download the binaries and follow the same steps as installing from binary releases
```
Comment (viccuad) [inline docs/user/tutorials/installing.md:64]: Nit:

```suggestion
On each successful CI run, artifacts are stored for that run. Just download the binaries and follow the same steps as installing from binary releases.
```

File: docs/user/tutorials/installing.md [hunk 7]
```diff
@@ -15,17 +53,29 @@ $ make
...
+
+---
```
Comment (viccuad) [inline docs/user/tutorials/installing.md:66]: Nit:
```suggestion
```

**Review by viccuad — APPROVED** [review]
Nice! Approving!

(omg I'm sorry for the amount of nitpicks :sweat_smile: , doc merging should be pleasant and not full bike-shedding, feel free to merge!)

File: docs/user/tutorials/installing.md (hunk 1 above)
Comment (viccuad) [inline docs/user/tutorials/installing.md:6]:
```suggestion
## From the binary beleases (Linux, macOS, windows)
```

Comment (viccuad) [reply docs/user/tutorials/installing.md:6]: (tried the GH "Add suggestion to batch".. sorry for the noise)

Comment (dragonchaser) [reply docs/user/tutorials/installing.md:6]: `s/Binary Releases/binary releases/` while we are at it :)

File: docs/user/tutorials/installing.md [hunk 8]
```diff
@@ -1,11 +1,49 @@
...
+
+```
+$ curl -fsSL -o get_hypper https://raw.githubusercontent.com/rancher-sandbox/hypper/main/scripts/get-hypper
+$ chmod 700 get_hypper
+$ ./get-hypper
+```
+
+Yes, you can curl https://raw.githubusercontent.com/rancher-sandbox/hypper/main/scripts/get-hypper | bash if you want to live on the edge.
+
+---
```
[the remark this answers is not in the document]
Comment (Itxaka) [reply docs/user/tutorials/installing.md:31]: This is here so it generate kind of a "box" around the note. Sorry didnt knew how to mark it properly so I thought that maybe two lines and a "title" could work as a note box.

File: docs/user/tutorials/installing.md (hunk 1 above)
Comment (Itxaka) [reply docs/user/tutorials/installing.md]: done!

File: docs/user/tutorials/installing.md (hunk 8 above)
[the remark this answers is not in the document]
Comment (Itxaka) [reply docs/user/tutorials/installing.md]: removed and added cursive + NOTE to see if its clearer

**Review by viccuad — APPROVED** [review]

**Pull request merged**